src/panel/plugin/meson.build

```diff
@@ -0,0 +1,61 @@
+libpanelpluginmanager_deps = [
+  link_libconfig,
+  link_libpanelplugin,
+  dep_glib,
+  dep_gobject_introspection,
+  dep_peas,
+]
+
+libpanelpluginmanager_sources = [
+  'plugin-manager.c',
+]
+
+libpanelpluginmanager_headers = [
+  'plugin-manager.h',
+]
+
+libpanelpluginmanager = static_library(
+  'panelpluginmanager',
+  libpanelpluginmanager_sources,
+  dependencies: libpanelpluginmanager_deps,
+  include_directories: [
+    include_directories('.'),
+  ]
+)
+
+link_libpanelpluginmanager = declare_dependency(
+  link_with: libpanelpluginmanager,
+  include_directories: include_directories('.')
+)
+
+# Generate .gir
```

Comment:
were we intending to offer an api to-do plugin management?  Maybe clean this bit up?

Reply:
Another thing I forgot to delete.